tests/test_actions.py

```diff
@@ -640,3 +654,181 @@ def test_describe(self) -> None:
+        q6 = QuietlyResolvable(FakeResolution())
+
+        assert isinstance(q1, Answerable)
+        assert isinstance(q2, Performable)
+        assert isinstance(q3, Resolvable)
+        assert isinstance(q4, Answerable)
+        assert isinstance(q5, Performable)
+        assert isinstance(q6, Resolvable)
+
+    def test_passthru_attribute(self):
+        a = FakeAction()
+        a.describe.return_value = "Happy Thoughts"
+        assert Quietly(a).describe() == "Happy Thoughts"
+
+    def test_passthru_attribute_missing(self):
+        a = FakeAction()
+        q = Quietly(a)
+        msg = "QuietlyPerformable(FakeAction) has no attribute 'desribe'"
+        with pytest.raises(AttributeError) as exc:
+            q.desribe()
+        assert str(exc.value) == msg
+
+    def test_answerable_answers(self, Tester):
+        question = FakeQuestion()
+        Quietly(question).answered_by(Tester)
+        question.answered_by.assert_called_once_with(Tester)
+
+    def test_performable_performs(self, Tester):
+        action = FakeAction()
+        Quietly(action).perform_as(Tester)
+        action.perform_as.assert_called_once_with(Tester)
+
+    def test_resolvable_resolves(self):
+        resolution = FakeResolution()
+        Quietly(resolution).resolve()
+        resolution.resolve.assert_called_once_with()
+
+
+class Action1(Performable):
+    @beat("{} tries to Action1")
+    def perform_as(self, actor: Actor):
+        actor.will(Quietly(Action2()))
+
+
+class Action2(Performable):
+    @beat("{} tries to Action2")
+    def perform_as(self, actor: Actor):
+        settings.DEBUG_QUIETLY = True
+        actor.will(Quietly(See(SimpleQuestion(), IsEqualTo(True))))
+
+
+class Action3(Performable):
+    @beat("{} tries to Action3")
+    def perform_as(self, actor: Actor):
+        actor.will(Quietly(Action4()))
+
+
+class Action4(Performable):
+    @beat("{} tries to Action4")
+    def perform_as(self, actor: Actor):
+        actor.will(Quietly(See(SimpleQuestion(), IsEqualTo(True))))
+
+
+class TestQuietlyDebug:
+    @pytest.fixture(autouse=True)
+    def setup(self):
+        settings.DEBUG_QUIETLY = False
+        yield
+        settings.DEBUG_QUIETLY = False
+
+    def test_kinking(self, Tester: Actor, mocker: MockerFixture):
+        mock_clear = mocker.spy(the_narrator, "clear_backup")
+        mock_flush = mocker.spy(the_narrator, "flush_backup")
+        mock_kink = mocker.spy(the_narrator, "mic_cable_kinked")
+
+        Tester.will(Quietly(FakeAction()))
+
+        assert mock_kink.call_count == 1
+        assert mock_clear.call_count == 2
+        assert mock_flush.call_count == 1
+
+    def test_skip_creation(self) -> None:
+        settings.DEBUG_QUIETLY = True
+        q = Quietly(FakeAction())
+        assert isinstance(q, FakeAction)
+
+    def test_debug_no_kink(self, Tester: Actor, mocker: MockerFixture) -> None:
+        mocker.patch.object(settings, "DEBUG_QUIETLY", return_value=True)
```

Comment:
How come this was needed here instead of just setting the `settings` attribute like in other tests?

Reply:
Fixed in latest commits